docs/troubleshooting/nodetool-memory-read-timeout.rst

```diff
@@ -16,7 +16,7 @@ Analysis
 ^^^^^^^^
 Nodetool is a Java based application which requires memory. ScyllaDB by default consumes 93% of the node’s RAM (for MemTables + Cache) and leaves 7% for other applications, such as nodetool.
 
-If cases where this is not enough memory (e.g. small instances with ~64GB RAM or lower), Nodetool may not be able to run due to insufficient memory. In this case an out of memory (OOM) error may appear and scylla-jmx will not run.
```

Comment:
I think this trobleshooting document is not valid anymore, since we moved to native nodetool which does not use Java (this issue description is talking about OOM on OpenJDK and JMX server).
So I think we should drop whole document, not just this line.